engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -127,7 +127,7 @@ <h4 translate>Basic Information</h4>
 
     <div class="detail" ng-show="repository.content_type == 'yum'">
       <span class="info-label" translate>Publish via HTTPS</span>
-      <span class="info-value" translate> true </span>
+      <span class="info-value" translate> Yes</span>
```

Comment:
Nitpick on something that was already there before, but there's an extra space before 'Yes'